sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -72,6 +72,11 @@
     <Compile Include="Shared\CancellationHelper.cs" />
     <Compile Include="Shared\OperationPoller.cs" />
     <Compile Include="Shared\TypeReferenceTypeAttribute.cs" />
+    <Compile Include="..\..\System.ClientModel\src\Internal\BitVector640.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\System.ClientModel\src\System.ClientModel.csproj" />
```

Comment:
This will become a package reference after System.ClientModel GA's, which is a prerequisite to merging this PR.